block-storage-attacher/tests/e2e/conf/developer.conf

```diff
@@ -43,13 +43,14 @@ IMAGE_REGISTRY=stg.icr.io
 USER_NAMESPACE=contsto2	# Read from environment
 PLUGIN_IMAGE=ibmcloud-block-storage-attacher
 DRIVER_BUILD=$PLUGIN_BUILD
+OPENSHIFT_INSTALL=$OPENSHIFT_INSTALL
 
 # Helm chart install
 KUBECONFIG=$KUBECONFIG			# Read from environment
 HELM_CHART_REPO=$HELM_CHART_REPO
 HELM_CHART=$GOPATH/src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/helm/ibm-block-storage-attacher/
 
-PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/portworx
+PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/community/portworx
```

Comment:
This helm chart path is wrong I guess. The current charts of portworx are at `https://github.com/IBM/charts/tree/master/community/portworx`. So need to use this location.